packages/node/src/evm/wallet.ts

```diff
@@ -1,16 +1,30 @@
 import { ethers } from 'ethers';
 import * as config from '../config';
 
-// 2^31-1 different wallets can be designated as below
-// m/0/0: masterWalletAddress
-// m/0/1: First reserved wallet path
-// m/0/2: Second reserved wallet path
-// ...
-// m/0/2^31-1 (each index is represented in 31-bits)
-// Although this can be extended as m/0/*/... it will not be needed.
-function getPathFromIndex(index: number | string) {
-  return `m/0/${index}`;
-}
+/**
+ * HD wallets allow us to create multiple accounts from a single mnemonic.
+ * Each sponsor creates a designated wallet for each provider to use
+ * in order for them to be able to respond to the requests their requesters make.
+ *
+ * By convention derivation paths start with a master index
+ * followed by child indexes that can be any integer up to 2^31.
```

Comment:
```suggestion
 * followed by child indices that can be any integer up to 2^31.
```

Thanks for adding the explanation here